script.js

```diff
@@ -161,19 +162,11 @@ function QuickWiki() {
               var dy = d.offset().top;
               var xgap = e.pageX - dx;
               var ygap = e.pageY - dy;
-              if (!$.browser.msie) {
-                e.preventDefault();
```

Comment:
Yes the default action of link clicking should be prevented from Quickwiki modal, not from the entire wiki page in general, as we already use Shift + Left click there. Will amend the code accordingly :)